api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
```

Comment:
Multiple update is weird already because DRF doesn't support it by default.  
The main issue here is that you have to match up which instance should be updated for each item in the list of validated_data.